bin/windows-env.ps1

```diff
@@ -47,11 +53,9 @@ $curlPkg = "${curlVer}-${mingwVer}"
 $Wix35_VERSION = '3.5.2519.20130612'
 
 $env:PATH = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path","User")
-if ($arch -eq 32) {
-  $env:PATH = "C:\tools\mingw32\bin;" + $env:PATH
-}
-$env:PATH += [Environment]::GetFolderPath('ProgramFiles') + "\Git\cmd"
+@([Environment]::GetFolderPath('ProgramFiles') + "\Git\cmd";,
```

Comment:
Looks like this semi-colon should be inside quotes. I can look into why that doesn't cause problems in a bit.